__tests__/workers/cdc.ts

```diff
@@ -282,6 +282,7 @@ describe('comment', () => {
     featured: false,
     createdAt: 0,
     lastUpdatedAt: 0,
+    setContentHTML: null,
```

Comment:
Why do you do that? I think it's better to fix the ChangeObject type to ignore functions in the given type